### ISSUE TITLE
Textures missing and spells doing nothing

### ISSUE THREAD

**Issue by Mineqwerty** [Growlith1223/ArsMagica2#349]
When I went to make a spell I noticed that the textures were all missing and the name randomizer didn't work. I figured it was fine, as long as i could use the spell. But I couldn't do damage with the projectile damage spell. Am i missing a patch?
![image](https://cloud.githubusercontent.com/assets/29109093/26658699/e44b26c8-4639-11e7-874d-5c09457d3b50.png)

**Comment by TheIcyOne** [comment]
Name randomizer's been broken for a while, since the site used for generation changed things up. Can we get your forge and AM2 versions?

**Comment by xenxero1** [comment]
seems like a simple enough process to do either natively, or through the server. Going through the server route opens up some fun options for customization in naming schemes at the discretion of the server owner.

**Comment by Mineqwerty** [comment]
Forge version: 1.10.2-forge 1.10.2-12.18.3.2185
AM2 version: 1.10.2-1.5.0-17

Thank you for helping

**Comment by TheIcyOne** [comment]
Yeah, there´s a lot of odd web-based stuff in the mod that could do with being rewritten at some point.

**Comment by Growlith1223** [comment]
Have you tried v18? coulda sworn i fixed this :T

**Comment by TheIcyOne** [comment]
I haven't seen this issue in v18, but I had difficulty reproducing it in older versions so it may just be me.
